idl/github.com/uber/cadence/shared.thrift

```diff
@@ -907,7 +913,14 @@ struct DescribeWorkflowExecutionRequest {
   20: optional WorkflowExecution execution
 }
 
+struct PendingActivityInfo {
+  10: optional string activityID
+  20: optional PendingActivityState state
+  30: optional binary heartbeatDetails
```

Comment:
If you exposing heartbeatDetails then I would also expose lastHeartbeatTimestamp.